src/engine-actions/is-node-config-valid.js

```diff
@@ -5,10 +5,19 @@ const { getInfo } = require('../lnd-actions')
  * hooked up to. Sets the engine validated flag to true if all validations pass.
  *
  * @function
- * @return {void}
+ * @throws {Error} Not Unlocked
+ * @throws {Error} No Chains Configured
+ * @throws {Error} Only allow, at most, one active chain
+ * @throws {Error} Mismatched Configuration
+ * @return {Boolean}
  */
-async function validateNodeConfig () {
-  const { chains = [] } = await getInfo({ client: this.client })
+async function isNodeConfigValid () {
```

Comment:
Renamed `validateNodeConfig` to `isNodeConfigValid` to fit in-line with our naming scheme for other engine actions.